sunpy/util/util.py

```diff
@@ -163,3 +164,34 @@ def dict_keys_same(list_of_dicts):
     for d in list_of_dicts:
         d.update({key: None for key in all_keys if key not in d})
     return list_of_dicts
+
+
+def hash_file(path):
```

Comment:
The example and/or docs page should call attention to this function! I didn't know it existed and used another CL tool to get the hashes.

Reply:
Quick online in the dev docs for it. I added it to the all above too so it should show up in the docs now.